src/io/flutter/view/FlutterViewAction.java

```diff
@@ -28,6 +29,14 @@
     this.app = app;
   }
 
+  @Override
+  public void actionPerformed(AnActionEvent e) {
```

Comment:
`final`?

Reply:
Nevermind, I think that would break `FlutterViewToggleableAction`.

Reply:
Exactly right.  (I tried that! 😄 )